Error: Cannot find module 'tmp' when using 0.16.0 on node 4

### Are you requesting a feature or reporting a bug?
Bug

### What is the current behaviour?

When running tests on travis using node 4, it fails with "Error: Cannot find module 'tmp'"

You can see more in full log of the run with 0.16.0 https://pastebin.com/QvzdTf6Q.

(Working version 0.15.0 is on https://pastebin.com/3zWw3PpK)

It also works on node 6.x and 8.x.

### What is the expected behaviour?

I can run tests using 0.16.0 on node 4.

### How would you reproduce the current behaviour (if this is a bug)?

Run tests in grunt-testcafe branch. See https://travis-ci.org/crudo/grunt-testcafe/jobs/244606414

#### Provide the test code and the tested page URL (if applicable)
n/a

### Specify your

* operating system: MacOS Sierra 10.12.5 or Ubuntu 14.04.5 LTS
* testcafe version: 0.16.0
* node.js version: v4.8.3

--- a/package.json
+++ b/package.json
@@ -109,6 +109,7 @@
     "testcafe-reporter-spec": "^2.1.1",
     "testcafe-reporter-xunit": "^2.1.0",
     "time-limit-promise": "^1.0.2",
+    "tmp": "0.0.28",
     "tree-kill": "^1.1.0",
     "typescript": "^2.2.2",
     "useragent": "^2.1.7"
@@ -163,7 +164,6 @@
     "saucelabs-connector": "^0.2.0",
     "serve-static": "^1.10.0",
     "strip-ansi": "^3.0.0",
-    "tmp": "0.0.28",
     "webmake": "0.3.42"
   }
 }

